site/_posts/2020-06-03-user-registration-and-sign-in-with-laravel.md

```diff
@@ -311,28 +361,28 @@ Now you can create a `profile.blade.php` file in your `resources/views/` directo
 ```
 {% endraw %}
 
-Any other user data you choose to store in FusionAuth will be available in the `session()->get('user')` array if you'd like to show it to logged in users.
+Any other user data you choose to store in FusionAuth will be available in the `session()->get('user')` array if you'd like to show it to logged-in users.
 
 When you log in, you should see a profile page that looks something like this:
 
-{% include _image.liquid src="/assets/img/blogs/fusionauth-laravel/fusionauth-2020-05-22-b.png" alt="The profile page for a Laravel application using FusionAuth." class="img-fluid" figure=false %}
+{% include _image.liquid src="/assets/img/blogs/fusionauth-laravel/profile-user-view.png" alt="The profile page for a Laravel application using FusionAuth." class="img-fluid bottom-cropped" figure=false %}
```

Comment:
Thanks for renaming the images!

Reply:
Does bottom-cropped do anything? I thought that was only in the docs css.

Reply:
It looks it does not (I assumed that it added margin, but that is already part of the `img` attribute).  I will remove the unneeded classes (looks like we have margin to spare) 👍🏻